src/views/v2/components/breadcrumbs/linked-breadcrumb.php

```diff
@@ -16,7 +16,7 @@
 ?>
 <li class="tribe-events-c-breadcrumbs__list-item">
 	<a
-		href="<?php echo esc_url( $breadcrumb['link'] ); ?>"
+		href="<?php echo esc_url( tribe_get_events_link() ); ?>"
```

Comment:
We should not be adjusting the template here. We set up the template variables in the BE, that's where we should be adjusting this value.